packages/build/tests/plugins/fixtures/feature_flags/manifest.yml

```diff
@@ -0,0 +1,2 @@
+name: test
+inputs: []
```

Comment:
I think you don't need the inputs here or?

Reply:
addressed in https://github.com/netlify/build/pull/4992/commits/d6429ab18dbfc6d3161d80e159ba5a0f9355e929